### DISCUSSION TITLE
docs/ninjutsu/low-ranking-ninja/at-workstation-but-dislike-it-just-focus-listen-music

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#22] [General]
# docs/ninjutsu/low-ranking-ninja/at-workstation-but-dislike-it-just-focus-listen-music

工作时的眼观六路、耳听八方是说：

https://996.ninja/docs/ninjutsu/low-ranking-ninja/at-workstation-but-dislike-it-just-focus-listen-music

**Comment by pocket20250310** [comment]
笑鼠 我一般都AppleMusic的催眠音乐 还不错

**Reply by MingGH** [reply]
在工位催眠质量一级棒
